lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<EOM
+
+'#{METADATA_REPOSITORY}' repository not found.
+Try running the command:
+  git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git ../#{METADATA_REPOSITORY}
```

Comment:
I just haven't used the https form for at least 5 or 6 years... so it is not something that I specifically would copy and paste.

I suspect that at this point, if they are getting to this file, they had to have written the corresponding `bin/generate` file, and so are probably not aware of what is required, so the documentation would be a good reference.

The purpose of the pull request, I think, was if the file doesn't exist, not the entire repository, so it is only one aspect, maybe, of what could be wrong?